recipes/pymks/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "pymks" %}
+{% set version = "0.3.4a7" %}
+{% set sha256 = "030d56481debaa4e68feee4102b1189e5f999c8aa427f4c431fa94c766f35f80" %}
+{% set hash = "ab2f93f57eeaf23b6d06d282dca60438c9a959e2ff97d0ecbdcf69acdfdf3a43" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/{{ hash[:2] }}/{{ hash[2:4] }}/{{ hash[4:] }}/{{ name }}-{{ version }}.tar.gz
```

Comment:
Please change to the "human" readable URL from the example. We rely on the re-direct and that is not a problem b/c we check with the `sha256` already.